plugins/starboard.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: starboard
+spec:
+  version: "v0.2.1"
+  homepage: https://github.com/aquasecurity/starboard
+  shortDescription: >-
+    Kubernetes-native security tool kit
```

Comment:
is there a more descriptive way to write this? here's my quick attempt based on your long description:

> Toolkit to find vulnerabilities in resources